crates/core_arch/src/aarch64/neon/generated.rs

```diff
@@ -2810,6 +2810,27 @@ pub unsafe fn vfmaq_f64(a: float64x2_t, b: float64x2_t, c: float64x2_t) -> float
     vfmaq_f64_(a, b, c)
 }
 
+/// Floating-point fused Multiply-Add to accumulator(vector)
+#[inline]
+#[target_feature(enable = "neon")]
+#[cfg_attr(test, assert_instr(fmadd))]
+pub unsafe fn vfma_n_f64(a: float64x1_t, b: float64x1_t, c: f64) -> float64x1_t {
+    let d: f64 = 0.0;
+    let e: float64x1_t = simd_insert(transmute(d), 0, c);
+    vfma_f64(b, e, a)
+}
+
+/// Floating-point fused Multiply-Add to accumulator(vector)
+#[inline]
+#[target_feature(enable = "neon")]
+#[cfg_attr(test, assert_instr(fmla))]
+pub unsafe fn vfmaq_n_f64(a: float64x2_t, b: float64x2_t, c: f64) -> float64x2_t {
+    let d: f64x2 = f64x2::new(0.0, 0.0);
+    let e: float64x2_t = simd_insert(transmute(d), 0, c);
+    let f: float64x2_t = simd_shuffle2(e, e, [0, 0]);
+    vfmaq_f64(b, f, a)
```

Comment:
This could be much simpler:
```suggestion
    let d: f64x2 = f64x2::new(c, c);
    vfmaq_f64(b, d, a)
```

Reply:
Done. Thanks. 

I get a new check error : vfmaq_n_f32_vfma' panicked at 'instruction found, but the disassembly contains too many instructions: #instructions = 26 >= 22 (limit). 

 I avoided checking this instruction.